packages/http-client-python/package.json

```diff
@@ -35,8 +35,6 @@
     "lint": "eslint . --max-warnings=0",
     "lint:py": "tsx ./eng/scripts/ci/lint.ts --folderName generator",
     "format": "pnpm -w format:dir packages/http-client-python && tsx ./eng/scripts/ci/format.ts",
-    "install": "tsx ./eng/scripts/setup/run-python3.ts ./eng/scripts/setup/install.py",
-    "prepare": "tsx ./eng/scripts/setup/run-python3.ts ./eng/scripts/setup/prepare.py",
```

Comment:
Why is "install" removed?

Reply:
They are moved to emitter, because we now only prepare Python environments for customers using venv to generate code. They are not required when using Pyodide.